charts/dependency/elasticsearch/values.yaml

```diff
@@ -0,0 +1,28 @@
+image:
+  repository: tccr.io/truecharts/elasticsearch
+  pullPolicy: IfNotPresent
+  tag: v8.10.2@sha256:0f4b76a1791bf9ca209773017266c28f5b1c8df3c2beb7ead8f7ce2a543f67e8
+
+securityContext:
+  container:
+    readOnlyRootFilesystem: false
+    runAsGroup: 0
+
+service:
+  main:
+    ports:
+      main:
+        port: 9200
+        targetPort: 9200
+
+persistence:
+  data:
+    enabled: true
+    mountPath: "/bitnami/mongodb"
```

Comment:
I'd suggest going over your PR!
This looks like its not tested at all.

Also, I don't see any inputs for password. (Yes ES has auth)

Reply:
Fixed that, also removed that persistence there too. Not sure how I missed that lol. Thanks for your help with this.

Reply:
??? So without persistent data, how does that work?

Reply:
> Fixed that, also removed that persistence there too. Not sure how I missed that lol. Thanks for your help with this.

The chart needs persistence, just copied the wrong one from another bitnami chart, look into it otherwise you lose everything in the container upon restarts

Reply:
Just fixed that, lol I have no idea why I removed that. The path from the docs says `/usr/share/elasticsearch/data` named `data` so I tossed that in there! What an epic fail :)